src/interfaces/docker_interface.py

```diff
@@ -130,8 +131,19 @@ def get_bitcoin_cli(self, tank: Tank, method: str, params=None):
             cmd = f"bitcoin-cli -regtest -rpcuser={tank.rpc_user} -rpcport={tank.rpc_port} -rpcpassword={tank.rpc_password} {method}"
         return self.exec_run(tank.container_name, cmd, user="bitcoin")
 
+    def get_file_from_container(self, container_name, path):
```

Comment:
```suggestion
    def get_file(self, container_name, file_path):
```

slight preference for breveity